coderd/database/dbtestutil/db.go

```diff
@@ -49,6 +50,12 @@ func WithDumpOnFailure() Option {
 	}
 }
 
+func WithReturnSQLDB(f func(*sql.DB)) Option {
```

Comment:
Non-blocker: Alternatively, we could extract a new function `dbtestutil.NewSQLDB(t testing.TB, opts ...Option) (database.Store, database.Pubsub, *sql.DB)`? I feel that would be more immediately obvious and less annoying to use.

Reply:
Also a good idea, I'll do that instead.

Reply:
ee86b3a